config/apPipe.py

```diff
@@ -1,10 +0,0 @@
-# Config override for lsst.ap.pipe.ApPipeTask
-import os.path
-from lsst.utils import getPackageDir
-
-decamConfigDir = os.path.join(getPackageDir('obs_decam'), 'config')
-
-# This default processCcd config assumes calibration products are from the
-# DECam Community Pipeline (CP). The config file to use for stack-built
-# calibs is "processCcd.py" instead.
-config.ccdProcessor.load(os.path.join(decamConfigDir, "processCcdCpIsr.py"))
```

Comment:
I don't think we can remove this file; we need to have an `apPipe.py` that calls `processCcd.py`.